tools/flang2/utils/ilitp/aarch64/ilitp.n

```diff
@@ -5388,6 +5465,107 @@ Half-precision min
 .AT arth comm hp cse
 .CG notCG
 
+.IL LDQP arlnk nme stc
+Load quad-precision floating value.  'stc' is not used.
+.AT load null qp
+.CG notCG
+
+.IL STQP qplnk arlnk nme stc
+Store quad precision quantity.  'stc' must be MSZ_F16.
+.AT store null trm
+.CG notCG
+
+.IL QADD qplnk qplnk
+quad-precision floating-point addition.
+.AT arth comm qp cse
+.CG notCG
+
+.IL QSUB qplnk qplnk
+quad-precision floating-point addition.
+.AT arth null qp cse
+.CG notCG
+
+.IL QMUL qplnk qplnk
+quad-precision multiply.
+.AT arth comm qp cse
+.CG notCG
+
+.IL QNEG qplnk
+quad-precision negate.
+.AT arth null qp cse
+.CG notCG
+
+.IL QDIV qplnk qplnk
+quad divide.
+.AT arth null qp cse
+.CG notCG
+
+.IL QMAX qplnk qplnk
+quad precision maximum.
+.AT arth comm qp cse
+.CG notCG
+
+.IL QMIN qplnk qplnk
+quad precison minimum.
+.AT arth comm qp cse
+.CG notCG
+
+.IL QFLOAT irlnk
+Integer to quad-precision conversion.
+.AT arth null qp cse
+.CG notCG
+
+.IL QFLOATK krlnk
+Integer64 to quad-precision conversion.
+.AT arth null qp cse
+.CG notCG
+
+.IL RQUAD splnk
+Single to quad-precision conversion.
+.AT arth null qp cse
+.CG notCG
+
+.IL DQUAD splnk
```

Comment:
Here should be `dplnk` instead of `splnk`.

Reply:
Fixed